src/Textarea/Textarea.tsx

```diff
@@ -70,59 +54,54 @@ export const Textarea = forwardRef(function Textarea(
     onBlur,
     rows = 4,
     required = false,
+    renderAsTitle = false,
   }: TextareaProps,
   ref: ForwardedRef<HTMLTextAreaElement>,
 ) {
   const id = useUniqueId(idProp)
   return (
-    <Box flex direction="column" className={className}>
-      {label && (
-        <Box mb="4px">
-          <Text tag="label" color="subtext" typo="label" htmlFor={id}>
-            {label}
-          </Text>
-          {required && (
-            <Text tag="label" color="error" typo="label">
-              *
-            </Text>
-          )}
-        </Box>
-      )}
-
-      <Box flex direction="column">
-        <Field
-          error={error}
-          id={id}
-          name={name}
-          disabled={disabled}
-          resize={resize}
-          placeholder={placeholder}
-          value={value}
-          onChange={(e: FormEvent<HTMLTextAreaElement>) => {
-            onChange && onChange(e.currentTarget.value)
-            onInputChange && onInputChange(e)
-          }}
-          maxLength={maxLength}
-          ref={ref}
-          onBlur={(e) => {
-            onBlur && onBlur(e)
-          }}
-          rows={rows}
-        />
-      </Box>
-      {error && <ErrorBox>{errorMsg}</ErrorBox>}
-    </Box>
+    <Field
+      className={className}
+      renderAsTitle={renderAsTitle}
+      id={id}
+      error={error}
+      label={label}
+      value={value}
+      errorMsg={errorMsg}
+      required={required}
+      fullHeight
+    >
+      <StyledTextArea
+        error={error}
+        id={id}
+        name={name}
+        disabled={disabled}
+        resize={resize}
+        placeholder={placeholder}
+        value={value}
+        onChange={(e: FormEvent<HTMLTextAreaElement>) => {
```

Comment:
This don't need to be typed as it is inferred by default
```suggestion
        onChange={(e) => {
```

Reply:
I don't think that's right - TS linter complains that e is implict any, without the typing.

Reply:
yeah I think that's why we had the type in the first place

Reply:
not on my machine 😅
![image](https://user-images.githubusercontent.com/33699873/176911488-9579a521-6c5f-4cdc-a40a-3bb585ca3f4a.png)

Reply:
Weird 🤔 The styled component itself should be rightfully typed so we shouldn't have any issue, what's the error message?